cmd/mt-control-server/controlmodels/index.go

```diff
@@ -0,0 +1,26 @@
+package controlmodels
+
+type IndexDelByQueryReq struct {
+	Expr      []string `json:"expr" form:"expr" binding:"Required"`
+	OlderThan int64    `json:"olderThan" form:"olderThan"`
```

Comment:
should this be called `From` to be consistent with IndexRestoreReq ? (it also seems to operate on LastUpdate)

Reply:
I was using the terminology from the `delByQuery` endpoint. At this point, it does seem to make more sense to use `From`/`To` since everything else uses that and `delByQuery` might be removed. That being said, I find `OlderThan` to be easier for my brain than `To`.